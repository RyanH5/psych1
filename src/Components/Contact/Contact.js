import React from 'react';

export const Contact = () => {
  return (
    <div>
      <h1>Contact Dr. Harrinton</h1>
      <h4><strong>Phone:</strong>612-766-9255</h4>
      <h4><strong>Fax:</strong>952-854-5062</h4>
      <h4><strong>Location:</strong></h4>
      <h5>8009 34TH AVE S, STE 1490</h5>
      <h5>BLOOMINGTON, MN 55425</h5>
      <p>
      If you are interested in psychotherapy or you are already in psychotherapy with me, I would appreciate you contacting me at 612-766-9255. Although e-mail is convenient, I ask that you contact me by phone because of concerns regarding confidentiality. I will return your call promptly.
      </p>
    </div>
  )
}