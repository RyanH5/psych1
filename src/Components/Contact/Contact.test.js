import React from 'react';
import { shallow } from 'enzyme';
import { Contact }from './Contact';

describe('Contact', () => {
  it('should match snapshot', () => {
    let wrapper = shallow(<Contact />)

    expect(wrapper).toMatchSnapshot();
  });
});