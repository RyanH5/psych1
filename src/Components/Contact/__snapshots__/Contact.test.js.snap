// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Contact Dr. Harrinton
</h1>,
        <h4>
          <strong>
                    Phone:
          </strong>
          612-766-9255
</h4>,
        <h4>
          <strong>
                    Fax:
          </strong>
          952-854-5062
</h4>,
        <h4>
          <strong>
                    Location:
          </strong>
</h4>,
        <h5>
          8009 34TH AVE S, STE 1490
</h5>,
        <h5>
          BLOOMINGTON, MN 55425
</h5>,
        <p>
          If you are interested in psychotherapy or you are already in psychotherapy with me, I would appreciate you contacting me at 612-766-9255. Although e-mail is convenient, I ask that you contact me by phone because of concerns regarding confidentiality. I will return your call promptly.
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Contact Dr. Harrinton",
        },
        "ref": null,
        "rendered": "Contact Dr. Harrinton",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <strong>
              Phone:
</strong>,
            "612-766-9255",
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Phone:",
            },
            "ref": null,
            "rendered": "Phone:",
            "type": "strong",
          },
          "612-766-9255",
        ],
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <strong>
              Fax:
</strong>,
            "952-854-5062",
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Fax:",
            },
            "ref": null,
            "rendered": "Fax:",
            "type": "strong",
          },
          "952-854-5062",
        ],
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <strong>
            Location:
</strong>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Location:",
          },
          "ref": null,
          "rendered": "Location:",
          "type": "strong",
        },
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "8009 34TH AVE S, STE 1490",
        },
        "ref": null,
        "rendered": "8009 34TH AVE S, STE 1490",
        "type": "h5",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "BLOOMINGTON, MN 55425",
        },
        "ref": null,
        "rendered": "BLOOMINGTON, MN 55425",
        "type": "h5",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "If you are interested in psychotherapy or you are already in psychotherapy with me, I would appreciate you contacting me at 612-766-9255. Although e-mail is convenient, I ask that you contact me by phone because of concerns regarding confidentiality. I will return your call promptly.",
        },
        "ref": null,
        "rendered": "If you are interested in psychotherapy or you are already in psychotherapy with me, I would appreciate you contacting me at 612-766-9255. Although e-mail is convenient, I ask that you contact me by phone because of concerns regarding confidentiality. I will return your call promptly.",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Contact Dr. Harrinton
</h1>,
          <h4>
            <strong>
                        Phone:
            </strong>
            612-766-9255
</h4>,
          <h4>
            <strong>
                        Fax:
            </strong>
            952-854-5062
</h4>,
          <h4>
            <strong>
                        Location:
            </strong>
</h4>,
          <h5>
            8009 34TH AVE S, STE 1490
</h5>,
          <h5>
            BLOOMINGTON, MN 55425
</h5>,
          <p>
            If you are interested in psychotherapy or you are already in psychotherapy with me, I would appreciate you contacting me at 612-766-9255. Although e-mail is convenient, I ask that you contact me by phone because of concerns regarding confidentiality. I will return your call promptly.
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Contact Dr. Harrinton",
          },
          "ref": null,
          "rendered": "Contact Dr. Harrinton",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <strong>
                Phone:
</strong>,
              "612-766-9255",
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Phone:",
              },
              "ref": null,
              "rendered": "Phone:",
              "type": "strong",
            },
            "612-766-9255",
          ],
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <strong>
                Fax:
</strong>,
              "952-854-5062",
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Fax:",
              },
              "ref": null,
              "rendered": "Fax:",
              "type": "strong",
            },
            "952-854-5062",
          ],
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <strong>
              Location:
</strong>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Location:",
            },
            "ref": null,
            "rendered": "Location:",
            "type": "strong",
          },
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "8009 34TH AVE S, STE 1490",
          },
          "ref": null,
          "rendered": "8009 34TH AVE S, STE 1490",
          "type": "h5",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "BLOOMINGTON, MN 55425",
          },
          "ref": null,
          "rendered": "BLOOMINGTON, MN 55425",
          "type": "h5",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "If you are interested in psychotherapy or you are already in psychotherapy with me, I would appreciate you contacting me at 612-766-9255. Although e-mail is convenient, I ask that you contact me by phone because of concerns regarding confidentiality. I will return your call promptly.",
          },
          "ref": null,
          "rendered": "If you are interested in psychotherapy or you are already in psychotherapy with me, I would appreciate you contacting me at 612-766-9255. Although e-mail is convenient, I ask that you contact me by phone because of concerns regarding confidentiality. I will return your call promptly.",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
